tests/flows/transparent_proxy_upgradeability.go

```diff
@@ -0,0 +1,200 @@
+package flows
+
+import (
+	"context"
+	"math/big"
+
+	erc20tokenhome "github.com/ava-labs/avalanche-interchain-token-transfer/abi-bindings/go/TokenHome/ERC20TokenHome"
+	erc20tokenremote "github.com/ava-labs/avalanche-interchain-token-transfer/abi-bindings/go/TokenRemote/ERC20TokenRemote"
+	"github.com/ava-labs/avalanche-interchain-token-transfer/tests/utils"
+	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
+	"github.com/ava-labs/subnet-evm/core/types"
+	"github.com/ava-labs/teleporter/tests/interfaces"
+	teleporterUtils "github.com/ava-labs/teleporter/tests/utils"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/log"
+	. "github.com/onsi/gomega"
+)
+
+func TransparentUpgradeableProxy(network interfaces.Network) {
```

Comment:
Can you add a description of the steps done in this test?

Reply:
updated